client/my-sites/plan-features/index.jsx

```diff
@@ -481,6 +563,53 @@ export class PlanFeatures extends Component {
 		} );
 	}
 
+	handleUpgradeClick( singlePlanProperties ) {
+		const {
+			displayJetpackPlans,
+			isInSignup,
+			selectedSiteSlug,
+			withDiscount,
+			onUpgradeClick: ownPropsOnUpgradeClick,
+		} = this.props;
+
+		const {
+			availableForPurchase,
+			cartItemForPlan,
+			isPrivateAndGoingAtomic,
+			planPath,
+			productSlug,
+		} = singlePlanProperties;
+
+		if ( ownPropsOnUpgradeClick && ownPropsOnUpgradeClick !== noop && cartItemForPlan ) {
```

Comment:
if it were a `noop`, wouldn't it be fine to allow it to be called?
Would `isFunction( ownPropsOnUpgradeClick ) && ...` suffice here?

Reply:
Ya, the issue is that it defaults to `noop` and this change does not override the prop via `connect` as it did before.